### ISSUE TITLE
2021/07/11/from_vim_plug_to_packer/

### ISSUE THREAD

**Issue by Ka-lip** [jdhao/jdhao.github.io#48]
Hello jdhao,

I guess this is just a small typo.
`lua require('plugins.lua')` should be `lua require('plugins')` since the former command asks to load a file in `plugins/lua.lua`.

Please correct me if I am wrong.

Thank you again for your posts and looking forward to learning more stuff from your blog.

Best regards,
Kalip

**Comment by jdhao** [comment]
Hi, Kalip, this is indeed a typo. It should be `lua require('plugins')` if you put it inside vim script, or `require 'plugins'` if you put it inside a lua file.

**Comment by Bintux** [comment]
Hi jdhao,

I got an error using the plugins.lua example as you have posted it on the blog. When checking your file in git I noticed that you set the fn variable prior which is missing on your blog code snipet.
I added the line "local fn = vim.fn" and then the error would got away and packer would actually be cloned.
Thank you,
Eike

**Comment by jdhao** [comment]
yeah, the snippet is used for reference, not meant to be used verbatim. You should refer to the complete script I linked in that post for a working script.

**Comment by NostraDavid** [comment]
I still had to run `:UpdateRemotePlugins` manually :(

Oh well...
